demo/get_started.ipynb

```diff
@@ -0,0 +1,119 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with Jupytext\n",
+    "\n",
+    "This small notebook shows you how to activate Jupytext in the JupyterLab\n",
+    "environment. We'll show you a few things that you can do with Jupytext and\n",
+    "a bit of what happens under the hood.\n",
+    "\n",
+    "**Note: to run this notebook locally, you need to first follow the Jupytext\n",
+    "installation instructions and activate the JupyterLab plugin. If you're on\n",
+    "Binder, it should already work.**\n",
+    "\n",
+    "## Enabling Jupytext in a new notebook\n",
+    "\n",
+    "This notebook is brand new - it hasn't had any special extra metadata added\n",
+    "to it. \n",
+    "\n",
+    "If we want Jupytext to save files in multiple formats automatically,\n",
+    "we can use the JupyterLab **command palette** to do so.\n",
+    "\n",
+    "* Click on the little 🎨 icon to the left\n",
+    "* Then type **`Jupytext`**. You should see a number of commands come up. Each\n",
+    "  one tells Jupytext to save the notebook in a different\n",
+    "  file format automatically.\n",
+    "* Select **Pair notebook with Markdown**\n",
+    "\n",
+    "That's it! If you have Jupytext installed, it will now save your notebook in\n",
+    "markdown format automatically when you save this `.ipynb` file.\n",
+    "\n",
```

Comment:
I would add something like 'in addition'. It is important to make it clear that Jupyter will continue to save the original notebook as well! Also, maybe we can mention that the two files really point to the same document, and that the Markdown file can be edited as well.